coalib/core/Bear.py

```diff
@@ -130,6 +132,21 @@ class Bear:
     CAN_FIX = set()
     ASCIINEMA_URL = ''
     BEAR_DEPS = set()
+    """
+        Attributes for a new bear.
+
+        :param section:
+            The section object where bear settings are contained.
+        :param file_dict:
+            The file-dictionary containing a mapping of filenames to the
+            according file contents.
+        :raises RuntimeError:
+            Raised when bear requirements are not fulfilled.
+    """
+    section = attr.ib(hash=False)
+    file_dict = attr.ib(hash=False)
+    _dependency_results = attr.ib(init=False, default=defaultdict(list),
+                                  hash=False)
```

Comment:
actually where's the difference over assigning those inside a custom `__init__`?